src/Education.UnitTests/Education.UnitTests.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net4.8</TargetFramework>
```

Comment:
We have a [common properties file](https://github.com/SonarSource/sonarlint-visualstudio/blob/master/src/SonarLint.Test.props) that sets common values that should be used by all test projects. One of the values it sets is `TargetFramework`, so you don't need to set it here.

You should just need:
```
  <Import Project="..\SonarLint.Test.props" />
```